### PR TITLE
[4] show result of checking for updates in CLI

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Summary of Changes

The CLI command for checking for updates doesn't currently report if it actually found any updates to apply! 

This PR changes the output so that it tells you if updates were found. 

### Testing Instructions

Run the command with and without old version of a Joomla 4 compatible plugin installed that has updates available. 

```
php cli/joomla.php update:extensions:check
```

### Actual result BEFORE applying this Pull Request

Same message regardless of the number of updates found or not found

Screenshot 2021-04-15 at 22 06 38

### Expected result AFTER applying this Pull Request

Different output if updates are found (its not currently within this command to actually tell you what they are... that can come later) 

Screenshot 2021-04-15 at 22 07 51

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on ca81f88fad4b654f9267bbda20e222b5d14c2ad0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33153">issues.joomla.org/tracker/joomla-cms/33153</a>.</sub>

File: libraries/src/Console/CheckUpdatesCommand.php [hunk 1]
```diff
@@ -55,9 +55,17 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
 		$cache_timeout = 3600 * (int) $component->getParams()->get('cachetimeout', 6);
 
 		// Find all updates
-		Updater::getInstance()->findUpdates(0, $cache_timeout);
-
-		$symfonyStyle->success('Finished fetching updates');
+		$ret = Updater::getInstance()->findUpdates(0, $cache_timeout);
+
```
Comment (richard67) [inline libraries/src/Console/CheckUpdatesCommand.php]:
```suggestion

```
Remove white space.

**Comment by richard67** [comment]
@PhilETaylor I've allowed myself to fix the PHP CS error for you.

**Comment by richard67** [comment]
I've restored the previous test result in the issue tracker since the commit which invalidated it was just a code style change.

File: libraries/src/Console/CheckUpdatesCommand.php [hunk 2]
```diff
@@ -55,9 +55,16 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
 		$cache_timeout = 3600 * (int) $component->getParams()->get('cachetimeout', 6);
 
 		// Find all updates
-		Updater::getInstance()->findUpdates(0, $cache_timeout);
-
-		$symfonyStyle->success('Finished fetching updates');
+		$ret = Updater::getInstance()->findUpdates(0, $cache_timeout);
+		if ($ret)
```
Comment (richard67) [inline libraries/src/Console/CheckUpdatesCommand.php:60]:
```suggestion

		if ($ret)
```
My mistake.

**Comment by particthistle** [comment]
I have tested this item :white_check_mark: successfully on 9f46bed359eff5e0db888d41db21aadb19f53962<br /><br />Test results in CLI matched the expected result.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33153">issues.joomla.org/tracker/joomla-cms/33153</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33153">issues.joomla.org/tracker/joomla-cms/33153</a>.</sub>

**Pull request merged**